samples/apps/copilot-chat-app/SKWebApi/Program.cs

```diff
@@ -48,9 +46,9 @@ public static void Main(string[] args)
 
     private static void AddServices(IServiceCollection services, ConfigurationManager configuration)
     {
-        /*builder.Services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
+        /*services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
```

Comment:
For any commented-out code that you want to remain in the file, please prepend a comment explaining the purpose and/or when you would want to enable the code.

Reply:
Removed